schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+
+CREATE TABLE animals(
+    id INT GENERATED ALWAYS AS IDENTITY, 
+    name TEXT, 
```

Comment:
- [ ] I think the name should be `varchar` with max length `255`, please kindly fix that 🙃